cmd/tle/tle.go

```diff
@@ -42,15 +43,19 @@ func run() error {
 	}
 
 	var src io.Reader = os.Stdin
-	if name := flag.Arg(0); name != "" && name != "-" {
-		f, err := os.OpenFile(name, os.O_RDONLY, 0600)
-		if err != nil {
-			return fmt.Errorf("failed to open input file %q: %v", name, err)
+	if rawInput := flags.RawInput; rawInput != "" && rawInput != "-" {
+		src = strings.NewReader(rawInput)
+	} else {
+		if name := flag.Arg(0); name != "" && name != "-" {
```

Comment:
unless I'm misunderstanding `name != "-"` here should be `name != "--"`, and there should be another branch for parsing normal stdin.
I'm not against having a raw input flag for the android/iOS scenario you detailed on the other comment, but it would be great for users to have a way to pass in stdin too (if it's not too much extra work!)

Reply:
I think it is correct. Just verified locally also you can still pass in stdin the same way 
`echo "ABC" | ./tle --encrypt -D 3s -o abc.txt`

Reply:
Gave it a test locally and it works fine for me! will run the build

Reply:
Gave it a test locally and it works fine for me! will run the build